packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.tsx

```diff
@@ -34,13 +35,29 @@ const WalletGradientBackground: React.FC<TProps> = ({
         return `wallets-gradient--${gradientCurrencyClassName}-${device}-${type}-${theme}`;
     };
 
+    const IconWrapper = () => (
+        <div className='wallets-gradient__icon-pattern'>
+            {[...Array(device === 'desktop' ? 100 : 60)].map((_, index) => (
```

Comment:
What is 100 and 60?

Reply:
the maximum number of icons visible set following the current design. In desktop it is 4 rows of 25 and in mobile/tablet it is 3 rows of 20

Reply:
Could you add that as a comment in the codebase?